quarkchain/evm/specials.py

```diff
@@ -281,6 +284,24 @@ def proc_deploy_root_chain_staking_contract(ext, msg):
     return create_contract(ext, new_msg, target_addr)
 
 
+# 3 inputs: (minter, token ID, amount)
+def proc_mint_mnt(ext, msg):
+    gascost = 3
```

Comment:
how much gas should we charge for updating token balance? @qizhou

Reply:
I think it should be the same as CALL, where the gas is calculated as following

# Creating a new account
            if op == "CALL" and not ext.account_exists(to) and (value > 0):
                extra_gas += opcodes.GCALLNEWACCOUNT
            # Value transfer
            if value > 0:
                extra_gas += opcodes.GCALLVALUETRANSFER

since the case that "to" should already exist, we probably only need GCALLVALUETRANSFER,  which is 9000.

BTW: Looks like proc_transfer_mnt()'s gas price is underprice based on the same logic here.